sssd_test_framework/utils/realmd.py

```diff
@@ -0,0 +1,100 @@
+"""Manage enrollment in realms."""
```

Comment:
That sounds funny, to make it like other files 

```Realmd management utilities. ```

Reply:
Nah, current one is good. This description is from `man realm`  page.

Reply:
Yes, the realmd command manages enrollment in realms, but the class you created, is a utility to manage realmd.  That's why it sounds funny to me.

Reply:
It's fine; things can sound funny. =D